packages/contracts/contracts/interfaces/v0.8.x/ISharedMinterMinPriceV0.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title This interface adds support for serial English auction minting.
```

Comment:
SEA -> Min price

Reply:
fixed in [93ed329](https://github.com/ArtBlocks/artblocks-contracts/pull/1649/commits/93ed329f51b2b5ae6e8253c778267e22bf40ea2d)